scripts/accumulator-mqtt.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/python
```

Comment:
Good addition to the testing suite! :)

Place have a look to the `install_scripts` target in root `makefile`. This script should be also installed the same way accumulator-server.py is.